activerecord/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+*   Running db:prepare no longer dumps database schema multiple times for multiple migrations.
```

Comment:
I think this changelog entry is not needed. We usually don't write bug fixes to the changelog.

Reply:
Sure thing!